metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage.py

```diff
@@ -299,58 +363,71 @@ def _get_upstream_lineage_info(
                 self.report.upstream_lineage[dataset_name] = [
                     u.dataset for u in upstream_tables
                 ]
-            return (
-                UpstreamLineage(
-                    upstreams=upstream_tables,
-                    fineGrainedLineages=sorted(
-                        finegrained_lineages, key=lambda x: (x.downstreams, x.upstreams)
-                    )
-                    or None,
-                ),
-                column_lineage,
+            return UpstreamLineage(
+                upstreams=upstream_tables,
+                fineGrainedLineages=sorted(
+                    finegrained_lineages, key=lambda x: (x.downstreams, x.upstreams)
+                )
```

Comment:
Not clear to me why we are removing "column_lineage" here

Reply:
This was a legacy snowflake_lineage code carried along to populate dataset's column lineage in custom properties. Not used anymore.